javaharness/java/arcs/android/impl/RuntimeSettingsAndroidJsImpl.java

```diff
@@ -0,0 +1,90 @@
+package arcs.android.impl;
+
+import arcs.api.RuntimeSettings;
+import com.google.auto.value.AutoValue;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.logging.Logger;
+import javax.inject.Inject;
+
+/** For Javascript-based Arcs runtime. */
+public final class RuntimeSettingsAndroidJsImpl implements RuntimeSettings {
+  // Equivalent to &log parameter
+  private static final String LOG_LEVEL_PROPERTY = "debug.arcs.runtime.log";
+  // Equivalent to &explore-proxy parameter
+  private static final String USE_DEV_SEVER_PROXY = "debug.arcs.runtime.use_alds";
```

Comment:
typo: s/USE_DEV_SEVER_PROXY/USE_DEV_SEVRER_PROXY

Reply:
haha, gonna fix it! Thanks.

Reply:
Done